### PR TITLE
Simonpals lvnetzzz 64 investigate work with mail notifications

### PR DESCRIPTION
**Pull request by simonpals**

### CODE REVIEW

File: TheaterSchedule.BLL/Services/GmailService.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+                // Note: since we don't have an OAuth2 token, disable
+                // the XOAUTH2 authentication mechanism.
+                client.AuthenticationMechanisms.Remove("XOAUTH2");
+
+                try
+                {
+                    // Note: only needed if the SMTP server requires authentication
+                    //lpt1.noreply@gmail.com is valid email, with permission for less secure devices
+                    await client.AuthenticateAsync("lpt1.noreply@gmail.com", "Lv-375.Net");
```
Comment (vtorkalo) [inline TheaterSchedule.BLL/Services/GmailService.cs]: do we nee continue AuthenticateAsync if previous step ConnectAsync was failed?
Why don't place them inside single try catch?

Comment (simonpals) [reply TheaterSchedule.BLL/Services/GmailService.cs]: But if ConnectAsync fails it is immediately returns false in the next catch section and we can read error status after that. All catch types are placed after apropriate method.

Comment (vtorkalo) [reply TheaterSchedule.BLL/Services/GmailService.cs]: which sense in Authenticate when connection was failed?

**Comment by simonpals** [comment]
Remove try-catch sections

**Pull request closed without merging**